apps/services/auth/ids-api/test/stubs/genericStubs.ts

```diff
@@ -1,13 +1,12 @@
 import { createNationalId } from '@island.is/testing/fixtures'
-import faker from 'faker'
+import { faker } from '@faker-js/faker'
 
 export type NameIdTuple = [name: string, id: string]
 
-export const getFakeNationalId = () =>
-  faker.helpers.replaceSymbolWithNumber('##########')
+export const getFakeNationalId = () => faker.string.numeric(10)
```

Comment:
I can guarantee that the return values from `faker.helpers.replaceSymbolWithNumber('##########')` and `faker.string.numeric(10)` are contextual the same.

That being said, I've noticed a second, custom national id generation function in your codebase which does indeed follow the Icelandic standard (starting with the number 7). Since this function didn't do this before, I keeped the implemenation as is.